src/main/blackbox/blackbox.c

```diff
@@ -1853,8 +1853,8 @@ STATIC_UNIT_TESTED void blackboxLogIteration(timeUs_t currentTimeUs)
                 writeGPSHomeFrame();
                 writeGPSFrame(currentTimeUs);
             } else if (gpsSol.numSat != gpsHistory.GPS_numSat
-                    || gpsSol.llh.lat != gpsHistory.GPS_coord[GPS_LATITUDE]
-                    || gpsSol.llh.lon != gpsHistory.GPS_coord[GPS_LONGITUDE]) {
+                       || gpsSol.llh.lat != gpsHistory.GPS_coord.lat
+                       || gpsSol.llh.lon != gpsHistory.GPS_coord.lon) {
```

Comment:
```suggestion
                || gpsSol.llh.lat != gpsHistory.GPS_coord.lat
                || gpsSol.llh.lon != gpsHistory.GPS_coord.lon) {
```

Reply:
It's aligned to  `} else if (`. This one is IMO fine. Aligning to `if (` is confusing